shared/templates/template_BASH_service_disabled

```diff
@@ -3,7 +3,22 @@
 # strategy = enable
 # complexity = low
 # disruption = low
-# Include source function library.
-. /usr/share/scap-security-guide/remediation_functions
+{{%- if init_system == "systemd" %}}
```

Comment:
what's the reason for the dash?

Reply:
@mpreisler / @matejak Also in #3058, you add a dash on both sides on all of the macros.

Reply:
It strips whitespace: http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control
The official documentation is not so great, but generally you want to have whitespace in jinja code, because it helps you to understand what's going on, but you don't want to have the result full of line breaks.